import_export/admin.py

```diff
@@ -261,50 +260,60 @@ def import_action(self, request, *args, **kwargs):
             except UnicodeDecodeError as e:
                 return HttpResponse(_(u"<h1>Imported file has a wrong encoding: %s</h1>" % e))
             except Exception as e:
-                return HttpResponse(_(u"<h1>%s encountered while trying to read file: %s</h1>" % (type(e).__name__, import_file.name)))
+                return HttpResponse(
+                    _(u"<h1>%s encountered while trying to read file: %s</h1>" % (type(e).__name__, import_file.name))
+                )
 
             # prepare kwargs for import data, if needed
             res_kwargs = self.get_import_resource_kwargs(request, form=form, *args, **kwargs)
             resource = self.get_import_resource_class()(**res_kwargs)
 
             # prepare additional kwargs for import_data, if needed
             imp_kwargs = self.get_import_data_kwargs(request, form=form, *args, **kwargs)
-            result = resource.import_data(dataset, dry_run=True,
-                                          raise_errors=False,
-                                          file_name=import_file.name,
-                                          user=request.user,
-                                          **imp_kwargs)
+            result = resource.import_data(
+                dataset, dry_run=True, raise_errors=False, file_name=import_file.name, user=request.user, **imp_kwargs
+            )
 
-            context['result'] = result
+            context["result"] = result
 
             if not result.has_errors() and not result.has_validation_errors():
+                confirm_form = self.get_confirm_import_form()
+                # If that flag is true, the file has sent will be processed without show the confirmation form
+                # In multi-container architecture, it's impossible to ensure where the file will be saved to be imported after data confirmation.
+                if os.getenv("PROCESS_WITHOUT_SHOW_CONFIRM_FORM", False):
```

Comment:
**issue:** Looking at the project's pattern, I believe we should avoid getting this kind of thing from env var and we should retrieve it from settings. I believe it'd something like this: 

```suggestion
                if getattr(settings, "PROCESS_WITHOUT_SHOW_CONFIRM_FORM", False):
```

Also, your addition on `docs/installation.rst` looks like it retrieves from project's settings, not from and env var.